config/application.rb

```diff
@@ -12,7 +12,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('app', 'mailers', 'concerns')
     config.time_zone = 'UTC'
 
-    config.middleware.use Rack::Attack unless Figaro.env.enable_load_testing_mode == 'true'
+    config.middleware.use Rack::Attack unless Figaro.env.disable_email_sending == 'true'
```

Comment:
The problem I'm trying to solve is to prevent accidentally sending emails via Mandrill, which can result in issues like https://github.com/18F/identity-private/issues/2046. By tying this to the `disable_email_sending` config, we can be warned that the server being used for load testing was not configured properly.

I'd love to hear any ideas for better ways to ensure that 1) only the PT server is used for load testing and 2) that load testing should raise an error if the server is not configured properly for load testing